README.md

```diff
@@ -92,6 +92,19 @@ var client = new OktaClient(new OktaClientConfiguration
 
 Hard-coding the Okta domain and API token works for quick tests, but for real projects you should use a more secure way of storing these values (such as environment variables). This library supports a few different configuration sources, covered in the [configuration reference](#configuration-reference) section.
 
+### Create a scoped client
+Create a client scoped to a specific context to specify a custom content type:
+``` csharp
+var client = new OktaClient(new OktaClientConfiguration
+{
+    OktaDomain = "https://{{yourOktaDomain}}",
+    Token = "{{yourApiToken}}"
+});
+
+var scopedClient = client.CreatedScoped(new RequestContext { ContentType = "my-custom-content-type" });
```

Comment:
Based on our conversation last time, should we rename `CreatedScoped` method to something like `CreateScopedClient`?

Reply:
I think yes, we should.  I'll update shortly.